devops/dependencies.json

```diff
@@ -40,47 +46,6 @@
       "root": "{ARCHIVE_ROOT}/comp/oclfpga/linux"
     }
   },
```

Comment:
The nice thing about staging is that we know the versions of the dependencies. How do we know what "latest" is?

Reply:
latest is a moving tag.
Exact versions can be found in a container build logs or from the inside of the container through package manager commands, like:
```
# apt list --installed | grep intel-igc-cm                                                                                        
intel-igc-cm/now 1.0.119 amd64 [installed,local]                               
```